iftk/helpers/eleven_stream.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import AsyncIterator
+from typing import Optional
+from elevenlabs.client import ElevenLabs
+
+
+async def ElevenStream(
```

Comment:
2 naming nits:

1) Let's call this file just `elevenlabs.py`

2) Functions should be snake_case
```suggestion
async def eleven_stream(
```